packages/back-end/src/jobs/updateExperimentResults.ts

```diff
@@ -109,18 +124,21 @@ export default async function (agenda: Agenda) {
 
 async function updateSingleExperiment(job: UpdateSingleExpJob) {
   const experimentId = job.attrs.data?.experimentId;
-  const orgId = job.attrs.data?.organization;
-  if (!experimentId || !orgId) return;
+  const context = job.attrs.data?.context;
+
+  if (!experimentId || !context) return;
 
-  const experiment = await getExperimentById(orgId, experimentId);
+  const { org } = context;
+
+  const experiment = await getExperimentById(org.id, experimentId);
```

Comment:
We can refactor this to take in a context or leave for later

Reply:
Same as above - I've got a separate PR that updates the `ExperimentModel` and all of it's methods [here](https://github.com/growthbook/growthbook/pull/2055).